src/main/java/uk/nhs/digital/emailadapter/schedulingtasks/EmailScheduledTask.java

```diff
@@ -0,0 +1,180 @@
+import lombok.extern.slf4j.Slf4j;
+import microsoft.exchange.webservices.data.core.ExchangeService;
+import microsoft.exchange.webservices.data.core.PropertySet;
+import microsoft.exchange.webservices.data.core.enumeration.misc.ExchangeVersion;
+import microsoft.exchange.webservices.data.core.enumeration.property.BasePropertySet;
+import microsoft.exchange.webservices.data.core.enumeration.property.WellKnownFolderName;
+import microsoft.exchange.webservices.data.core.enumeration.search.LogicalOperator;
+import microsoft.exchange.webservices.data.core.enumeration.service.ConflictResolutionMode;
+import microsoft.exchange.webservices.data.core.service.item.EmailMessage;
+import microsoft.exchange.webservices.data.core.service.item.Item;
+import microsoft.exchange.webservices.data.core.service.schema.EmailMessageSchema;
+import microsoft.exchange.webservices.data.core.service.schema.ItemSchema;
+import microsoft.exchange.webservices.data.credential.ExchangeCredentials;
+import microsoft.exchange.webservices.data.credential.WebCredentials;
+import microsoft.exchange.webservices.data.property.complex.Attachment;
+import microsoft.exchange.webservices.data.property.complex.FileAttachment;
+import microsoft.exchange.webservices.data.property.complex.ItemAttachment;
+import microsoft.exchange.webservices.data.property.complex.MessageBody;
+import microsoft.exchange.webservices.data.search.FindItemsResults;
+import microsoft.exchange.webservices.data.search.ItemView;
+import microsoft.exchange.webservices.data.search.filter.SearchFilter;
+import microsoft.exchange.webservices.data.search.filter.SearchFilter.SearchFilterCollection;
+import uk.nhs.digital.emailadapter.service.StagedStopwatch;
+import uk.nhs.digital.emailadapter.transformer.PDFTransformer;
+
+@Slf4j
+@EnableAsync
+@Getter
+@Setter
+@Component
+public class EmailScheduledTask {
+
+  private ExchangeService service = new ExchangeService(ExchangeVersion.Exchange2010_SP2);
+  private AWSSimpleSystemsManagement ssm = AWSSimpleSystemsManagementClientBuilder.defaultClient();
+  StandardPBEStringEncryptor decryptor = new StandardPBEStringEncryptor();
+  private String mpCryptoPassword;
+
+  @Autowired
+  public EmailScheduledTask(@Value(value = "${mpCryptoPassword}") final String mpCryptoPassword)
+      throws Exception {
+    this.mpCryptoPassword = mpCryptoPassword;
+    decryptor.setPassword(mpCryptoPassword);
+    ExchangeCredentials credentials = new WebCredentials(
+        decryptor.decrypt(getParameter("username")), decryptor.decrypt(getParameter("password")));
+    service.setCredentials(credentials);
+    service.autodiscoverUrl(decryptor.decrypt(getParameter("username")));
+  }
+
+  @Async
+  @Scheduled(fixedRateString = "${fixedRate.in.milliseconds}",
+      initialDelayString = "${initialDelay.in.milliseconds}")
+  public void sendMails() throws InterruptedException {
+    String timeStamp =
+        DateTimeFormatter.ofPattern("yyyy-MM-dd_HH:mm:ss").format(LocalDateTime.now());
```

Comment:
Extract formatter out to class level. eg. DateTimeFormatter.ofPattern("yyyy-MM-dd_HH:mm:ss");

and use LocalDateTime.format(DateTImeFormatter) method instead.